Core/Systems/Affixes/ItemTypes/ArmorAffixes/DefenseAffixes.cs

```diff
@@ -1,52 +1,41 @@
 ﻿namespace PathOfTerraria.Core.Systems.Affixes.ItemTypes.ArmorAffixes;
 
-internal class DefenseAffixes
+internal class DefenseItemAffix : ItemAffix
 {
-	internal class DefenseItemAffix : ItemAffix
+	public override void ApplyAffix(EntityModifier modifier, PoTItem item)
 	{
-		public override ItemType PossibleTypes => ItemType.Armor;
-		public override void ApplyAffix(EntityModifier modifier, PoTItem item)
-		{
-			modifier.Defense.Base += 1 + Value * 5f + item.ItemLevel / 50f;
-		}
+		modifier.Defense.Base += 1 + Value * 5f + item.ItemLevel / 50f;
```

Comment:
Werent these supposed to use the data from the tier?

Reply:
From what I understand, `Value` would use the data from the tier, so it already does.

Reply:
Oh, so we still have some arbitrary formula but value range will depend on the affix tier, sure.

Reply:
Naw the value should not be evaluated against ItemLevel at all anymore. It should be STRICTLY the value rolled within the Tier. So this needs to be resolved.